monocle/webapp.py

```diff
@@ -150,14 +161,6 @@ def indices():
 
 
 def main():
-    config_path = os.getenv('CONFIG', None)
```

Comment:
Why not keeping this as a function. I'd like to have the authorization support when I run the flask dev server outside of docker or w/o the WSGI server

Reply:
I needed to have it called when loaded in all the cases. I don't understand what will be the advantage of having it in a function.

Reply:
ok that's fine yes.